main.go

```
@@ -33,9 +35,13 @@ func main() {
 			err := client.PostComment(pr, ghComment)
 			if err != nil {
 				fmt.Printf("Failed to post comment: %v\n", err)
-				fmt.Printf("Comment: %v\n", ghComment)
+				// TODO post comment to file.
 			}
 		}
+		err := evaluateComments(pr, client)
```

Bug: evaluateComments should return an error instead of log.Fatalf

---

Thank you for your feedback! The AI gives your comment a score of 4.